target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -416,6 +416,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "1:lan" "2:wan" "6@eth0"
 		;;
+	wizfi630s)
```

Comment:
Change it to `wiznet,wizfi630s` as specified in DTS.

Reply:
if i change it this way - the switch disappears. Idk how to resolve this, instead of keeping this line as it is.

Reply:
Found it - switch is fine again - forgot to remove the line in ramips.sh